README.md

```diff
@@ -30,14 +30,18 @@ This repository contains code for pneumonia detection using X-ray images of the
         - GoogleNet
         - Inception
     - If using colab for training, mount the drive and use --colab to save the files in the drive
+    - For faster training and Image augmentation, if on Linux, NVIDIA DALI can be used
+        - Install NVIDIA DALI
+        - Use --nvidiadali flag to use it
+        - Note that NVIDIA DALI is only available for Linux
     - To change optimizers, use one of the following
         - Adam
         - SGD
         - RMSprop
         - Adagrad
         - Adadelta
 - To test new dataset, run `test.py` with directory path and base model with option given above. Make sure that model is trained on those models first. 
-    - Example, `python3 test.py "./data/Corona_Classification_data/test/" Inception`
+    - Example, `python3 test.py "./data/Corona_Classification_data/test/" Inception True True`
```

Comment:
Can you also explain what those two `True` are and what they do?

Oh, never mind: That ar explained in the usage below!